README.md

```diff
@@ -41,17 +41,28 @@ tests in the `local_node-private` configuration.
 
 ## Docker
 
-Tests can be run with Docker with the following commands:
+Tests can be run with Docker with the following commands: 
 
 ```
 docker build . -t ceramic-integration-tests
 ```
 
 ```
 docker run \
-  -e DISCORD_WEBHOOK_URL='<url>' \
+  -e DISCORD_WEBHOOK_URL_TEST_RESULTS='<url_for_results>' \
```

Comment:
Having these env vars in alphabetical order would be helpful here too

Reply:
Alphabetical ordering does not always make sense, sometimes grouping in context provide more clarity. I will try my best effort to accommodate here. Updated